types/youtube-dl/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for youtube-dl 1.12
+// Project: https://www.npmjs.com/package/youtube-dl
+// Definitions by: Bogdan Surai <https://github.com/bsurai>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import 'node';
+import * as fs from "fs";
+
+type callback = (info: object) => void;
```

Comment:
This doesn't deserve its own type alias, just write `on(event: string, func: (info: object) => void): this;` inline.